src/main/java/uk/gov/hmcts/divorce/solicitor/event/page/Applicant2SolAosCosts.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.hmcts.divorce.solicitor.event.page;
+
+import uk.gov.hmcts.divorce.ccd.CcdPageConfiguration;
+import uk.gov.hmcts.divorce.ccd.PageBuilder;
+import uk.gov.hmcts.divorce.common.model.Applicant;
+import uk.gov.hmcts.divorce.common.model.CaseData;
+
+public class Applicant2SolAosCosts implements CcdPageConfiguration {
+
+    @Override
+    public void addTo(PageBuilder pageBuilder) {
+        pageBuilder
+            .page("Applicant2SolAosCosts")
+            .pageLabel("Paying for the divorce")
+            .label("LabelRespSol-AOSCost",
+                "The petitioner asked the court that the respondent pays some or all of the costs of the divorce.\n\n"
```

Comment:
worth checking if we should refer this as applicant 1 and applicant 2 rather  than petitioner and respondent

Reply:
Confirmation -  it should be applicant 1 & respondent